news/8179.bugfix

```diff
@@ -0,0 +1 @@
+Update mechanism to set permissions for files with execute permissions while extracting sdists.
```

Comment:
It's not a bugfix, the behaviour is the same. I'd label this `trivial`.

Reply:
The news entry has been labelled as `trivial`